homeassistant/components/hyundai_kia_connect/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""The Hyundai / Kia Connect integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import HyundaiKiaConnectDataUpdateCoordinator
+
+PLATFORMS: list[str] = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Set up Hyundai / Kia Connect from a config entry."""
+    """Set up this integration using UI."""
+    if hass.data.get(DOMAIN) is None:
+        hass.data.setdefault(DOMAIN, {})
+
+    coordinator = HyundaiKiaConnectDataUpdateCoordinator(hass, config_entry)
+    await coordinator.async_refresh()
+    hass.data[DOMAIN][config_entry.entry_id] = coordinator
```

Comment:
```suggestion
    hass.data.setdefault(DOMAIN, {})[config_entry.entry_id] = coordinator
```

Reply:
I think this is less readable, but do not have strong opinion